data/presets/traffic_sign/hazard/dangerous_junction.json

```diff
@@ -0,0 +1,23 @@
+{
+    "fields": [
+        "maxspeed/advisory"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "traffic_sign": "hazard",
+        "hazard": "dangerous_junction"
+    },
+    "reference": {
+        "key": "hazard",
+        "value": "dangerous_junction"
+    },
+    "aliases": [
+        "Dangerous Intersection Sign",
+        "Dangerous Junction Sign",
+        "Intersection Sign"
+    ],
+    "name": "Junction Warning Sign"
```

Comment:
I’d think that, in American English, “junction” would imply that the cross street is a numbered route. “Dangerous intersection sign” would be more accurate as a primary name for this preset.